javascript/javascript3/week1/Weather_app/weather-app.js

```diff
@@ -0,0 +1,108 @@
+// Using: Async Await
+
+//Select the city, click Todays weather to get weather data
+const getData = document.querySelector("#search");
+
+// Click Current Location Button, to get latitude and longitude
+const currentLocationBtn = document.querySelector("#current-location")
+console.log(currentLocationBtn)
+
+// All display variables
```

Comment:
These could be simplified by an object created from an array of the class names and a for loop where the keys are the class names and the values are the elements in question. For example if an array of the class names like `const classNames = ["location", "weather-image", ...]` exist. You can make an object with:
```
const displayElements = {};
for (const className of classNames)
    displayElements[className] = document.getElementsByClassName(className);
```

Reply:
Should I make this change now.... Or is it just for next time.

Reply:
Up to you, I am just suggesting a simpler way to solve the same problem. Otherwise, it is fine as long as it works.